### PR TITLE
fix(docs): Side bar navigation not accessible on small screens

### PR DESCRIPTION
**Pull request by tmorehouse**
The sidebar is not accessible when screen are XS size.

fixes issue #944

### CODE REVIEW

File: docs/nuxt/layouts/docs.vue [hunk 1]
```diff
@@ -5,6 +5,7 @@
         <div class="container-fluid">
             <div class="row flex-xl-nowrap2">
                 <div class="col-12 col-md-3 col-xl-2 bd-sidebar">
+                    <div class="d-block d-sm-none py-4 mt-3"></div>
```
Comment (tmorehouse) [inline docs/nuxt/layouts/docs.vue]: Temporary hack to push search & toggler in view on XS screens

Comment (mosinve) [reply docs/nuxt/layouts/docs.vue]: no positive effect

Comment (mosinve) [reply docs/nuxt/layouts/docs.vue]: image
toggler showed, but its not clickable

Comment (tmorehouse) [reply docs/nuxt/layouts/docs.vue]: ah may need to add v-b-toggle directive...

Comment (tmorehouse) [reply docs/nuxt/layouts/docs.vue]: My mistake.  I had component `<nav-toggle>` instead of `<b-nav-toggle>`

Comment (tmorehouse) [reply docs/nuxt/layouts/docs.vue]: Note search currently doesn't work from what I can see.  Its just there for when search will be applied.

I wonder if that should be hidden, or disabled?

**Comment by tmorehouse** [comment]
Is the toggle working correctly now?

**Pull request merged**